src/components/KanaQuiz/KanaQuiz.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import { KanaType } from '../../types/kana';
```

Comment:
You could also import/export types explicitly, like `import type { KanaType } from ...` or `import { type KanaType } from ...`.
There are some edge-case benefits listed below, but I also find it easier to just see at the imports which ones are types.

* https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html
* https://stackoverflow.com/questions/61412000/do-i-need-to-use-the-import-type-feature-of-typescript-3-8-if-all-of-my-import